awx/api/views/__init__.py

```diff
@@ -1422,7 +1425,7 @@ class CredentialInputSourceSubList(SubListCreateAPIView):
     parent_key = 'target_credential'
 
 
-class CredentialTypeExternalTest(SubDetailAPIView):
+class CredentialTypeExternalTest(GenericAPIView):
```

Comment:
if at all possible, it would be nice to keep this as the `SubDetailAPIView` and get everything to work by using the default model access permission class.

Reply:
:man_shrugging: works for me, I'll give it a try.